nifi-commons/nifi-security-socket-ssl/pom.xml

```diff
@@ -31,5 +31,17 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-security-utils</artifactId>
+            <version>1.14.0-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+            <version>4.1.63.Final</version>
```

Comment:
The latest version of  netty-handler appears to be 4.1.65.Final (May, 2021).

Reply:
Thanks for the feedback @markobean, I updated the version.